README.md

```diff
@@ -1 +1,61 @@
-# eoslime-shape-react
\ No newline at end of file
+# eoslime-shape-react
```

Comment:
Because of the image, make a new folder 'docs' or something like that and put images there so the Readme could have a way to read the images

Reply:
folder 'docs' added